src/utils/workspace.ts

```diff
@@ -26,14 +26,14 @@ export async function parseWorkspaceConfig(files : vscode.Uri[]) {
 		}
 	}
 
-	const content = await fileRead(files[0].fsPath);
+	const content = await fs.readFile(files[0].fsPath);
```

Comment:
With this replaced, I think we can delete the entire `fs.ts` file, no?

Reply:
yes, and it is removed as part of this PR already.

Reply:
Sorry, I didn't notice. 🙈